packages/expo-video/src/VideoView.types.ts

```diff
@@ -22,6 +22,13 @@ export interface VideoViewProps extends ViewProps {
    */
   nativeControls?: boolean;
 
+  /**
+   * Determines whether the player allows switching to external playback mode.
```

Comment:
it might be worth clarifying (in parentheses) that this is related to AirPlay, as someone coming from a non-iOS  background might not immediately be aware about it. You can also add a link to the official docs: https://developer.apple.com/documentation/avfoundation/avplayer/1387441-allowsexternalplayback

Reply:
Updated